Cargo.toml

```diff
@@ -44,6 +48,7 @@ uuid = { version = "1.6.1", features = ["v4"] }
 tracing = "0.1.40"
 tracing-subscriber = { version = "0.3.16", features = ["json", "env-filter", "ansi"] }
 env_logger = "0.10.1"
+log = "0.4.17"
```

Comment:
Previously we was remover `log` crate in favor of `tracing`. Can use `tracing` in tests or `log` is required for them?

Reply:
did it, thanks